js/symbol/resolve_text.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 var resolveTokens = require('../util/token');
+var TwitterCLDR = require('../../node_modules/twitter_cldr/min/en.min.js');
```

Comment:
don't rely on node_modules path, you can't just do `require('twitter_cldr/min/en.min.js')`